roles/openshift_sanitize_inventory/tasks/main.yml

```diff
@@ -85,5 +85,5 @@
       Ensure that template_service_broker_remove and template_service_broker_install are mutually exclusive,
       do not set both to true. template_service_broker_remove defaults to true.
   when:
-    - template_service_broker_remove | default(false) | bool
```

Comment:
This role does not have a defaults file.  We should either leave these two lines, or add the appropriate values in defaults/main.yml

Reply:
Thanks, I've reverted openshift_sanitize_inventory back to its original state.